### ISSUE TITLE
TS Playground does not provide ES2017 Object types

### ISSUE THREAD

**Issue by saschanaz** [microsoft/TypeScript#33902]
**TypeScript Version:**  Playground

**Search Terms:** playground es2017

**Code**

```ts
Object.entries({}) // Property 'entries' does not exist on type 'ObjectConstructor'.
```

**Expected behavior:** It should get no error.

**Actual behavior:** It does get one.

**Playground Link:** https://www.typescriptlang.org/play/?ssl=1&ssc=18&pln=1&pc=1#code/PIIwVgpgxgLgdBAdjATgSwgZwBQG8C+AlEA

**Related Issues:** 

Couldn't find a relevant place to post an issue. @orta, is it being maintained on https://github.com/orta/typescript-play ?

**Comment by jcalz** [comment]
Playground issues are found at [microsoft/TypeScript-Website](https://github.com/microsoft/TypeScript-Website/issues) (maybe the new issue template here should be updated to indicate that?)

**Comment by saschanaz** [comment]
Thanks! Could someone transfer this to there? 🙏
